File: programmers/튜플.java
import java.util.*;
class Solution {
    public int[] solution(String s) {
       int[] answer = {};
        int length=s.length();
        s= s.substring(1,s.length()-1);
        s = s.replace("}", "$");
        s = s.replace("{","$");
        s = s.replace(",","$");
        StringTokenizer stk = new StringTokenizer(s,"$",false);
        HashMap<String,Integer> map = new HashMap<>();
        while(stk.hasMoreTokens()){
            String token = stk.nextToken();
            map.put(token,map.getOrDefault(token,0)+1);
        }
        answer= new int[map.size()];
        for(String str : map.keySet()){
            answer[map.size()-map.get(str)]= Integer.parseInt(str);
        }
        return answer;
    }
}

File: programmers/튜플.py
def solution(s):
    answer = []
    s = s[2:-2]
    s = s.split('},{')
    s = sorted(s, key=lambda x: len(x))
    
    answer.append(s[0])
    
    for i in range(1, len(s)):
        s[i] = s[i].split(',')
        for j in s[i]:
            if j not in answer:
                answer.append(j)
    answer = list(map(int, answer))
    return answer
